.github/workflows/connector-publish.yml

```diff
@@ -0,0 +1,97 @@
+name: Connector Publish workflow
+
+permissions: read-all
+
+on:
+  workflow_call:
+    inputs:
+      package-name:
+        required: true
+        type: string
+      branch:
+        required: true
+        type: string
+        default: "main"
+      rust-profile:
+        type: string
+        default: "release"
+      cloud-url:
+        required: true  
+        type: string
+    secrets:
+      CLOUD_USER_EMAIL:
+        required: true
+      CLOUD_USER_PASSWORD:
+        required: true
+
+jobs:
+  linux:
+    name: linux
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
this should be probably made as parameter and made as true for default

Reply:
Done